crazyflie_server_py/crazyflie_server_py/crazyflie_server.py

```diff
@@ -8,7 +8,7 @@
 from cflib.crazyflie.swarm import CachedCfFactory
 from cflib.crazyflie.swarm import Swarm
 
-from crazyswarm2_interfaces.srv import Takeoff, Land, GoTo
+from crazyflie_interfaces.srv import Takeoff, Land, GoTo
```

Comment:
You should probably also change this line in the same file.

`get_package_share_directory("crazyswarm2"), "config", "crazyflies.yaml"
`

Reply:
wait... maybe this is easier to fix on my PR. I predict merge conflicts 😄  Could you perhaps revert this change to the original and I'll fix it as soon as possible with mine? I guess it should be fine as few knows of it existence yet

Reply:
I fixed it in this PR, just so that the history makes sense. Since the changes are very local, there should be no merge conflict (or they should be super easy to resolve).